apps/docs/vite.config.ts

```diff
@@ -3,6 +3,13 @@ import { sveltepress } from "@sveltepress/vite";
 import { defineConfig } from "vitest/config";
 
 export default defineConfig({
+	resolve: {
```

Comment:
This is temporary, right? Until we solve #47

Reply:
This is a good question! Actually, it’s required because `svelte2tsx` uses `node:path` (see https://github.com/sveltejs/language-tools/blob/master/packages/svelte2tsx/src/svelte2tsx/index.ts#L11), so we need this polyfill.

(Since node:path is just a small library, I think using UnJS’s [pathe](https://github.com/unjs/pathe) as a replacement is a reasonable choice.)

Reply:
Ah, so because of `svelte2tsx` we will still need this one.

It can be solved if we decide to `svelte2tsx` with our own parser. I've described this case there: https://github.com/svelte-docgen/svelte-docgen/discussions/17#discussioncomment-11577524
I'd like to add TODO comment with reference to this discussion and _when it can be removed_, so it's likely we will forget in the future as to _why_ we have it.

Reply:
Thanks! I’ve added a TODO comment.  (I completely agree about not relying on svelte2tsx.)